essence-of-live-coding/test/Cell/Util.hs

```diff
@@ -192,4 +212,98 @@ test =
           }
     ]
 
+type TestTraversables' = TestTraversables '[Maybe, [], V.Vector 10, Seq, Map Int]
```

Comment:
Can you put everything related to this in a separate module `Cell.Util.Traversable`